Article/Article.js

```diff
@@ -3,16 +3,17 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
```

Comment:
```suggestion
    this.expandButton = this.domElement.querySelector('.expandButton');
```
Since on line 6 you created a reference to domElement with this.domElement, I would be sure to use it. Both ways work.